contracts/RelayHub.sol

```diff
@@ -403,17 +404,25 @@ contract RelayHub is IRelayHub {
 
         atomicData.preReturnValue = abi.decode(retData, (bytes32));
 
+        bool abortOnFirstRevert = true;
+        (uint count, bytes memory error) = CallBatcher.sendBatch(
```

Comment:
commenting out the call to `sendBatch` makes the compilation work.
the same call exists in the TestBatch.sol, so not sure why it fails the compiler here.